src/database/network-table.c

```diff
@@ -1581,103 +1581,87 @@ void updateMACVendorRecords(void)
 		dbclose();
 }
 
-char *__attribute__((malloc)) getDatabaseHostname(const char *ipaddr)
+// Get hardware address of device identified by IP address
+char *__attribute__((malloc)) getMACfromIP(const char *ipaddr)
```

Comment:
This is not correct, `git` screwed up the `diff`. I removed the `getDatabaseHostname()` function altogether as (almost) the same is done by `getNameFromIP()` below.